### ISSUE TITLE
Use "=" in curly syntax for html attribute declarations

### ISSUE THREAD

**Issue by ericscheid** [naturalcrit/homebrewery#1488]
The new curly format looks like this: `{{classname,#idname,cssproperty:cssvalue CONTENT GOES HERE}}`.

**Proposal: have the equals character `=` signify `htmlattribute=attributevalue`.**

This would apply to the braced-span, braced-block, and braced-injector variants.

---
#### injecting the `ismap` attribute onto the `img` element in this link:

```
[![alt text](https://images.io/file.png "Title attribute"){ismap=true}](https://mappy.io/)
```

#### setting the lang attribute for a block of text

```
{{lang="fr",quote
Quand on a pas ce que l’on aime, il faut aimer ce que l’on a.
}}
```

**Comment by ericscheid** [comment]
This particular feature might be a while off, but in the meantime can we disallow the use of `=` in the curly declarations so as to reserve its use for this?

**Comment by 5e-Cleric** [comment]
Fixed by #3131
